lang/gnu-apl/Portfile

```diff
@@ -28,6 +28,12 @@ checksums           rmd160  e277ae100c8eccae32980b606401b04bfe65b0b1 \
                     sha256  144f4c858a0d430ce8f28be90a35920dd8e0951e56976cb80b55053fa0d8bbcb \
                     size    3565018
 
+<<<<<<< Updated upstream
+=======
+livecheck.type      regex
+livecheck.regex     apl-(\[0-9.\]+)${extract.suffix}
+
+>>>>>>> Stashed changes
```

Comment:
You broke this.

Reply:
Sorry, apparently I'm not very good at git.